lib/components/result-view/history.js

```diff
@@ -47,6 +47,9 @@ const History = observer(({ store }: { store: OutputStore }) => {
         style={{
           fontSize: atom.config.get(`Hydrogen.outputAreaFontSize`) || "inherit"
         }}
+        hydrogen-wrapOutput={Boolean(
```

Comment:
Why is this `Boolean` casting needed ? The type of the config value seems to come as `boolean`

Reply:
For both flow, and for some reason when running the spec atom.config is apparently undefined so it cast undefined to false for specs.

Reply:
I see.

And I prefer just `wrapOutput` rather than `hydrogen-wrapOutput` since it's just an attribution like `style`, `onClick` etc.

Reply:
The only reason I prefixed it was because of setting standard because what if someone unknowing later uses a common attribute or what if another package uses our same technique. I just have it there so there is no possible confusions.

Reply:
I'm not sure about that because `hydrogen-wrapOutput` is not a class name and thus if an user's stylesheet or the other packages's one has defined the style for that, I think this won't get impact by that.
But maybe I miss some point, and it's a small thing thus you can ignore the recommendation.

Reply:
The point is if someone else's package has a wrapoutput attribute and we have a wrapoutput attribute behaviors could get screwy because attribute aren't for css only they can be accessed from js. Of course this means another package would have to have a wrapoutput which is unlikely, but it is just a precaution.